skimage/morphology/convex_hull.py

```diff
@@ -77,7 +77,7 @@ def _check_coords_in_hull(gridcoords, hull_equations, tolerance):
     return coords_in_hull
 
 
-def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10):
+def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10, return_labels=False):
```

Comment:
I understand what you're saying. What do you think about providing the actual `inside_labels` as a parameter to the `convex_hull_image`? 

By default, this parameter can be `inside_labels = [1, 2, 3]` by default. It would give more flexibility to the user 🧐

Reply:
Hum... This assumes that the user knows the strategy behind the scene of `convex_hull_image` and that he knows the `grid_point_in_poly_label` function and the definitions of different labels... I am not sure that this is super "user friendly" :confused: 
The only direct application of this that I can think of is the ability of drawing the convex hull borders by setting `inside_labels` to `[2, 3]`, which is kind of paradox to remove inside point label from `inside_labels` :sweat_smile: ...
If such an output is of any interest, a dedicated function (`convex_hull_border_image`??) is maybe better suited, but this may be addressed in a separate PR :wink:

Reply:
Okay, I'm convinced 🙂
I changed my PR to use `include_borders`, as you suggested. Let me know what you think!